src/methods/defining_methods/hard/q4/GreaterThan.java

```diff
@@ -0,0 +1,32 @@
+package methods.defining_methods.hard.q4;
+
+/**
+ * [J] Write a Java Program that asks the user for two integers, then prints out the greater of the two integers.
+ * Write a method called greaterNum that accepts two integers as parameters and returns the greater of the two.
```

Comment:
Add something about that the main method should print the larger returned integer